libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -919,7 +923,7 @@ Vector3f AC_AttitudeControl::euler_accel_limit(const Vector3f &euler_rad, const
     } else {
         rot_accel.x = euler_accel.x;
         rot_accel.y = MIN(euler_accel.y / cos_phi, euler_accel.z / sin_phi);
-        rot_accel.z = MIN(MIN(euler_accel.x / sin_theta, euler_accel.y / sin_phi), euler_accel.z / cos_phi);
+        rot_accel.z = MIN(MIN(euler_accel.x / sin_theta, euler_accel.y / (sin_phi * cos_theta)), euler_accel.z / (cos_phi * cos_theta));
```

Comment:
Just want to double check if I need to do any divide by zero checks here. As long as we get an INF and MIN will take the real number it should be fine.

Reply:
should be no need thanks to the constrains on 913 to 916.